evalml/model_understanding/graphs.py

```diff
@@ -1526,7 +1526,8 @@ def get_linear_coefficients(estimator, features=None):
             "before using this estimator."
         )
     coef_ = estimator.feature_importance
-    coef_ = pd.Series(coef_, name="Coefficients", index=features)
+    coef_ = pd.Series(coef_, name="Coefficients")
+    coef_.index = features
```

Comment:
I couldn't get it to work with them together, because of pandas behavior so it reindexes after and returns all NaNs if put on the same line (or that's my rough understanding)?https://pandas.pydata.org/docs/reference/api/pandas.Series.html